AK-16248 Update connector-aws-vpc doucmentation (#17)
alkiranet/terraform-provider-alkira: docs/resources/connector_aws_vpc.md

Manage AWS Cloud Connector.

> Either `vpc_cidr` or `vpc_subnet` needs to be specified for routing
> purpose. If `vpc_cidr` is provided, it will select all associcated
> subnets of VPC. Otherwise, you could select certain subnets by
> specifying `vpc_subet`.
> 
> `vpc_route_tables` could be used to adjust the routing options against
> the specified route tables. If `OVERRIDE_DEFAULT_ROUTE` is specified,
> the existing default route will be overrided and the traffic will be
> routed to Alkira CXP. If `ADVERTISE_CUSTOM_PREFIX` is specified, you
> could specify a list of prefixes for which traffic must be routed to
> Alkira CXP.

Reviewer: ```suggestion
Either `vpc_cidr` or `vpc_subnet` needs to be specified for routing
purpose. If `vpc_cidr` is provided, it will automatically select all associated
subnets of the given VPC. Otherwise, you could select certain subnets by
specifying `vpc_subnet`.

`vpc_route_tables` could be used to adjust the routing options against
the specified route tables. When `OVERRIDE_DEFAULT_ROUTE` is specified,
the existing default route will be overwritten and the traffic will be
routed to Alkira CXP. When `ADVERTISE_CUSTOM_PREFIX` is specified, you
need to provide a list of prefixes for which traffic must be routed to
Alkira CXP.
```

## Limitations

There are several limitations of AWS connector so far:

* Changing an existing connector to a new AWS VPC is not supported at
this point. You need to create a new connector for a new AWS VPC.

> * Updating an existing connector may need to re-provision to make the
>   change effective, e.g. changing the segment the connector is
>   associated.

Reviewer: ```suggestion
* Updating an existing connector might require the tenant network to be re-provisioned to make the
  change effective, e.g. changing the segment the connector is
  associated.
```

- **cxp** (String) The CXP where the connector should be provisioned.
- **name** (String) The name of the connector.
- **segment** (String) The segment of the connector belongs to. Currently, only `1` segment is allowed.

> - **size** (String) The size of the connector, one of `SMALL`, `MEDIUM`, `LARGE` or `2LARGE`.

Author: Right now, only AWS-VPC supports 2LARGE. 4LARGE support is not fully ready yet.
